app/assets/javascripts/components/student/student_edit.es6.jsx

```diff
@@ -50,13 +61,19 @@ class StudentEdit extends Component {
     };
   }
 
+  editCard() {
+    this.props.renderOverlay()
+  }
+
   render() {
     return (
       <div style={this.styles.container}>
         <h5>{this.renderTitle()}</h5>
-        <i
-          className={'fa fa-pencil-square-o'}
-          style={this.styles.icon} />
+        <Clickable
+          func={this.editCard.bind(this)}
```

Comment:
Instead of creating a new function called `editCard`, we can just do `(event) => this.props.renderOverlay()`!